src/lib/components/CreatorOrContributorField.js

```diff
@@ -74,7 +74,7 @@ const _CreatorOrContributorField = ({ field, form, ...props }) => {
     form.setFieldValue(nameFieldPath, `${givenName} ${familyName}`);
   };
 
-  const personValue = 'Personal';
+  const isPerson = getIn(form.values, typeFieldPath, null) === 'Personal'
```

Comment:
I would rename it probably to `isPersonalType`. I think it's better to give the exact meaning of this variable, WDYT?

Reply:
Ah, I think in this case the *exact meaning* is that a creator/contributor is a person or an organization. The fact that we use "Personal" or "Organizational" as the *value* to indicate that is a detail: we could have used "person"/"organization" or "p"/"o" or any other symbol. And if we do change that symbol, we don't have to change the variable name because it would still reflect the meaning. It's then easier to read: the abstraction makes the code easier to understand while not lying.

Reply:
Ah ok, I took the meaning of the value for granted :) Ok then it's ok with me!